lib/JSON/Validator/OpenAPI/Mojolicious.pm

```diff
@@ -0,0 +1,72 @@
+package JSON::Validator::OpenAPI::Mojolicious;
+
+use Carp qw(confess);
+use Mojo::Base 'JSON::Validator::OpenAPI';
+
+### request ###
+
+sub _get_request_data {
+  my ($self, $c, $in) = @_;
+
+  return $c->req->url->query->to_hash(1)  if $in eq 'query';
+  return $c->match->stack->[-1]           if $in eq 'path';
+  return $c->req->body_params->to_hash(1) if $in eq 'formData';
+  return $c->req->headers->to_hash(1)     if $in eq 'header';
```

Comment:
Why are you passing in a true value to the to_hash() methods? The old code did not do this, and it breaks swagger2.pm header tests.